### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by robinjo78**
Splits the code from the VideoPane class into several other classes to have a better overview of the implementation of the various Panes. The classes created are:
- MenuMediaPane which is split into MenuPane and MediaPane
- MediaBar
- TimeLoggerPane

### CODE REVIEW

**Comment by egedebruin** [comment]
I think it would be better to create a pane package for the panes in the gui package, because the gui package is a little big now.

**Comment by robinjo78** [comment]
@egedebruin done!

File: src/main/java/gui/panes/MediaPane.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ * Class that constructs the MediaPane for the VideoPane.
+ */
+public class MediaPane {
+    /**
+     * Class parameters.
+     */
+    private FlowPane mediaPlayerPane;
+
+    /**
+     * Constructor for MenuPane.
```
Comment (lisv) [inline src/main/java/gui/panes/MediaPane.java]: Spelling mistake, 'MenuPane' should be MediaPane.

File: src/main/java/gui/panes/VideoPane.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+     */
+    public Pane createVideoPane(final Stage primaryStage) {
+        BorderPane videoPane = new BorderPane();
+
+        ArrayList<Pane> mmp =
+            menuMediaPane.createMenuAndMediaPane(videoPane, primaryStage);
+
+        // get the menubar and put it at the top of the videoPane
+        videoPane.setTop(mmp.get(0));
+        // get the imageView (location where videos are shown)
```
Comment (lisv) [inline src/main/java/gui/panes/VideoPane.java]: Comment is outdated, it should say: "imageViews"

**Pull request merged**